### DISCUSSION TITLE
Everything is not swell - april incidents diary

### DISCUSSION THREAD

**Discussion by bafomet88** [swellstores/community#236] [Help] [answered]
April was disaster concerning swell stability, I would like everyone to realize, how many hours swell was unstable / unresponsive, because I think that status page is not showing the correct numbers, even with comparison to swell official announcements.

I count hours below on the basis of official announcements, I compare it with status page.

06.04 API issues affecting cart and checkout
major outrage, duration about 12 hours (sic!). No any "sorry" from the company. 
According to official status page - 5h

11.04 Hosted storefront partial outage
duration about 5h
According to official status page - 0.42min

15.04 Backend API degraded performance
duration about 5/6h for some users
According to official status page - 0.21min

24.04 Backend API degradation
duration: 5h
According to official status page - not even mention, everything was swell ;/

27.04 Backend API timeouts
duration: 3h
According to official status page - not even mention, everything was swell ;/

To sum up:
swell had issues for about 31 hours in April, what seems unbelievably high number, but this was the fact. In the meantime, company did not offered any compensation. Additionally, their status page is worth nothing - data there is nowhere near the real statistics. 

I don't know what to think about this policy - I understand that mistakes happens, but as a customer the data above shows clearly, that company is hiding it's problems. That is why I decided to write this post, I hope, that it will influence company policy toward us as a users.

**Comment by ericingram** [comment, marked as answer]
@bafomet88 it's true that our process for updating status page has not been great, and we have been working on improvements. In general we are going to start adding a status update when *any* relevant report comes in, which wasn't been part of the process before.

We will retroactively update the page (although I can't claim to know if your exact numbers are correct yet).
